rs/src/resources/substrait-version

```diff
@@ -0,0 +1 @@
+v0.2.0
```

Comment:
Why not use something like [`git-version`](https://crates.io/crates/git-version)?

Since the version will be `const` the binary has it baked in and you shouldn't need to store a file and duplicate the version information.

Reply:
> Why not use something like git-version?

I forget what all the problems were exactly, but git repo information need not necessarily exist during the build. Note also that this is the Substrait version, not the validator version; the submodule certainly won't exist unless a user is compiling manually from within the repo. It's not included by `cargo package` or in Python source distributions, and it would be silly to force them to do so, including history, and also expect that the user has `git` on their path (Windows...), just to get a version tag from the current checkout of a submodule.

> Since the version will be `const` the binary has it baked in and you shouldn't need to store a file and duplicate the version information.

I'm not sure what you mean by this. The binary has it built-in because this (source) file exists. Binary distributions don't include the file (well, I'm not 100% on that, but they don't need to).